src/labthings_fastapi/__init__.py

```diff
@@ -0,0 +1,32 @@
+from .thing import Thing
+from .descriptors import ThingProperty, ThingSetting
+from .decorators import (
+    thing_property,
+    thing_setting,
+    thing_action,
+    fastapi_endpoint,
+)
+from . import deps
+from . import outputs
+from .outputs import blob
+from .server import ThingServer
+
+# The symbols in __all__ are part of our public API.
+# They are imported when using `import labthings_fastapi as lt`.
+# We should check that these symbols stay consistent if modules are rearranged.
+# The alternative `from .thing import Thing as Thing` syntax is not used, as
+# `mypy` is now happy with the current import style. If other tools prefer the
+# re-export style, we may switch in the future.
+__all__ = [
+    "Thing",
+    "ThingProperty",
+    "ThingSetting",
+    "thing_property",
+    "thing_setting",
+    "thing_action",
+    "fastapi_endpoint",
+    "deps",
+    "outputs",
+    "blob",
+    "ThingServer",
```

Comment:
Needed for !296 of microscope

```suggestion
    "ThingServer",
    "cli",
    "get_blocking_portal",
```